src/workload.rs

```diff
@@ -257,8 +259,26 @@ impl WorkloadManager {
         config: config::Config,
         metrics: Arc<Metrics>,
         awaiting_ready: readiness::BlockReady,
+        cert_manager: Box<dyn CertificateProvider>,
     ) -> anyhow::Result<WorkloadManager> {
-        let workloads: Arc<Mutex<WorkloadStore>> = Arc::new(Mutex::new(WorkloadStore::default()));
+        let (tx, mut rx) = mpsc::channel::<Identity>(100);
```

Comment:
lets make it 256, this is the max pods per node on typical k8s cluster.

Reply:
good idea